### DISCUSSION TITLE
v0.21.1

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#3302] [Versions & Releases]
- Adding a basic Hindi text cleaner. https://github.com/coqui-ai/TTS/commit/32065139e713b3e44aa88e72c4d35012bb888238

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.21.1'>v0.21.1</a>.</em>

**Comment by tahercoolguy** [comment]
Adding basic hindi text cleaner means you adding Hindi language ?
